### ISSUE TITLE
I cannot change SQLite GUI font size from preference

### ISSUE THREAD

**Issue by CeciliaWJH** [sqlitebrowser/sqlitebrowser#2166]
Details for the issue
--------------------

#### What did you do?
I cannot find a place to change the GUI font size. (NOT editor font size, NOT log file font size)

#### What did you expect to see?
Bigger GUI words.

#### What did you see instead?
![image](https://user-images.githubusercontent.com/48337701/77219520-f86e9e00-6b0c-11ea-83f8-51b549e691df.png)

I can barely read the word in the blue circle.
Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [V] Windows: ( _version:_ windows 10 64 bits___ )
- [ ] Linux: ( _distro:_ ___ )
- [ ] macOS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [V] 3.11.2
- [ ] 3.11.1
- [ ] 3.10.1
- [] Other: ___

#### Did you also

- [V] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [V] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by scottfurry** [comment]
@justinclift - I suspect this is apart of Qt's "appearance" settings. On Linux, this can only be controlled via the [qt5ct|https://sourceforge.net/projects/qt5ct/]. This tweaks the underlying stylesheet for all qt5 apps. HDMI presentation also factors into this problem.

**Comment by justinclift** [comment]
Yeah, I'm pretty sure other people have reported this exact problem before too.

Not remembering off hand how (if) it was solved for them though, and kind of busy trying to get my head around Rust (the programming lang) atm.

**Comment by CeciliaWJH** [comment]
@scottfurry I tried to install qt5ct but I can't, I guess it's because I'm not using Linux but Windows 10.
NMV, thank you @scottfurry and @justinclift . It you know any update to this issue, please let me know.  :)

**Comment by mgrojo** [comment]
@CeciliaWJH Try with the latest release v3.12. If that does not solve the problem, use a [nightly build](https://nightlies.sqlitebrowser.org/latest/), which has a new preference setting to control the general font size.
